src/core/SceneObjectBase.tsx

```diff
@@ -82,7 +82,7 @@ export abstract class SceneObjectBase<TState extends SceneObjectState = SceneObj
   /**
    * Subscribe to the scene state subject
    **/
-  public subscribeToState(observerOrNext?: Partial<Observer<TState>>): Subscription {
+  public subscribeToState(observerOrNext?: Partial<Observer<TState>>): SubscriptionLike {
```

Comment:
Does "OrNext" mean something?
```suggestion
  public subscribeToState(observer: Partial<Observer<TState>>): SubscriptionLike {
```

Reply:
observerOrNext the next part is from how subscribe method signatures look like insisde rxjs (where they take an observer or a next function), but this function does not take a next function (I wish it did), but can add that later.